spec/policies.yml

```diff
@@ -102,7 +102,7 @@ components:
           ##### Permissions required
 
           `create` privilege on the policy."
-        operationId: "updatePolicy"
+        operationId: "append"
```

Comment:
```suggestion
        operationId: "appendPolicy"
```

Reply:
Resolved in conversation during standup. Dropping the Policy because of class names in generated clients